pages/samvera/developer_community/release_testing.md

```diff
@@ -0,0 +1,105 @@
+---
+title: How to Coordinate Testing for a Release
+a-z: [Testing, Release Process]
+last_updated: March 30, 2017
+tags: [release, community]
+sidebar: samvera_sidebar
+permalink: release_testing.html
+folder: samvera
+---
+
+<br />
+<br />
+
+Each release will have a release testing coordinator. This person can vary from release to release. You will work closely with the Hyrax Tech Lead and Product Owner.
+
+Follow the process outlined in [Hyrax Testing and Release Process](https://wiki.duraspace.org/display/samvera/Hyrax+Testing+and+Release+Process). As this is a new process, you may find that some sections need adjustment or more detail.
```

Comment:
This will also be a page from our site so link should refer to our page.

Reply:
Thanks @laritakr

Reply:
Hey I have a few edits to my own writing :). 1. Can you  change "new process" to "newly defined process"?

Reply:
Yes.